corehq/apps/hqadmin/pillow_settings.py

```diff
@@ -104,3 +104,21 @@ def dump_yaml(pillow_configs):
     print 'Excluded Pillows'
     pillow_configs = list(_get_pillow_configs_from_settings_dict(pillows_by_group))
     print dump_yaml(sorted(set(pillow_configs) - set(pillows)))
+
+
+def get_pillow_env_config(environment):
+    pillow_conf = {}
+    pillow_file = os.path.join(
+        'deployment/commcare-hq-deploy/fab',
```

Comment:
this would be some setting in the dev settings that would point to your deploy repo